app/views/account/weeklyDetails.scala.html

```diff
@@ -0,0 +1,60 @@
+@import com.gu.memsub.BillingSchedule
+@import com.gu.memsub.subsv2.SubscriptionPlan
+@import com.gu.memsub.subsv2.Subscription
+@import model.DigitalEdition.UK
+@import com.gu.salesforce.Contact
+@import com.gu.i18n.Country
+@import controllers.ManageWeekly.WeeklyPlanInfo
+@import model.SubscriptionOps._
+@import org.joda.time.LocalDate
+@(
+    subscription: Subscription[SubscriptionPlan.WeeklyPlan], billingSchedule: BillingSchedule, contact: Contact, billToCountry: Country, plans: List[WeeklyPlanInfo]
+)(implicit r: RequestHeader, touchpointBackendResolution: services.TouchpointBackend.Resolution)
+
+@main("Your Guardian Weekly subscription | The Guardian", bodyClasses = List("is-wide"), edition = UK, touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
+    @helper.javascriptRouter("jsRoutes")(
+        routes.javascript.Promotion.validate
+    )
+    <script>
+
+        guardian.plans = [
```

Comment:
this is just passing to the react form so it knows what to renew people onto so we problably don't need it when we're not allowing renewals